pkg/isoeditor/stream.go

```diff
@@ -29,7 +47,69 @@ func NewRHCOSStreamReader(isoPath string, ignitionContent *IgnitionContent, ramd
 		return nil, err
 	}
 
-	r, err := readerForFileContent(isoPath, ignitionImagePath, isoReader, ignitionReader)
+	// Starting with version 0.17.0 of the CoreOS installer a new '/coreos/igninfo.json' file may
+	// exist to indicate the location of the ignition inside the ISO. For example, in the S390
+	// platform it will contain something like this:
+	//
+	//	{
+	//		"file": "images/cdboot.img",
+	//		"offset": 66497660,
+	//		"length": 262144
+	//	}
+	//
+	// If it exists we need to read it, otherwise we can use the '/images/ignition.img' location
+	// used in previous versions.
+	var ignInfo IgnInfo
```

Comment:
Suggesting so separate the code that get the `ignInfo` into a different function `NewRHCOSStreamReader` is already long enough

Reply:
+1

Reply:
I will do that if we decide to use this instead of Nick's #139.